content/zh/docs/tasks/traffic-management/request-timeouts/index.md

```diff
@@ -11,93 +11,188 @@ test: yes
 
 本任务用于示范如何使用 Istio 在 Envoy 中设置请求超时。
 
+{{< boilerplate gateway-api-gamma-support >}}
+
 ## 开始之前 {#before-you-begin}
 
 * 按照[安装指南](/zh/docs/setup/)中的说明安装 Istio。
 
 * 部署示例应用程序 [Bookinfo](/zh/docs/examples/bookinfo/)，
-  并应用[默认目标规则](/zh/docs/examples/bookinfo/#apply-default-destination-rules)。
-
-* 运行以下命令初始化应用的版本路由：
-
-    {{< text bash >}}
-    $ kubectl apply -f @samples/bookinfo/networking/virtual-service-all-v1.yaml@
-    {{< /text >}}
+  包括[服务版本](/zh/docs/examples/bookinfo/#define-the-service-versions)。
 
 ## 请求超时 {#request-timeouts}
 
-HTTP 请求的超时可以用[路由规则](/zh/docs/reference/config/networking/virtual-service/#HTTPRoute)的
-**timeout** 字段来指定。默认情况下，超时是禁用的，本任务中，会把 `reviews` 服务的超时设置为 1 秒。
+HTTP 请求的超时可以通过路由规则中的超时字段来指定。
+默认情况下，超时是禁用的，本任务中，会把 `reviews` 服务的超时设置为半秒。
 为了观察效果，还需要在对 `ratings` 服务的调用上人为引入 2 秒的延迟。
 
-1. 将请求路由到 `reviews` 服务的 v2 版本，它会发起对 `ratings` 服务的调用：
-
-    {{< text bash >}}
-    $ kubectl apply -f - <<EOF
-    apiVersion: networking.istio.io/v1alpha3
-    kind: VirtualService
-    metadata:
-      name: reviews
-    spec:
-      hosts:
-        - reviews
-      http:
-      - route:
-        - destination:
-            host: reviews
-            subset: v2
-    EOF
-    {{< /text >}}
-
-1. 给对 `ratings` 服务的调用添加 2 秒的延迟：
-
-    {{< text bash >}}
-    $ kubectl apply -f - <<EOF
-    apiVersion: networking.istio.io/v1alpha3
-    kind: VirtualService
-    metadata:
-      name: ratings
-    spec:
-      hosts:
-      - ratings
-      http:
-      - fault:
-          delay:
-            percent: 100
-            fixedDelay: 2s
-        route:
-        - destination:
-            host: ratings
-            subset: v1
-    EOF
-    {{< /text >}}
-
-1. 在浏览器中打开 Bookinfo 的网址 `http://$GATEWAY_URL/productpage`。
+1) 将请求路由到 `reviews` 服务的 v2 版本，它会发起对 `ratings` 服务的调用：
```

Comment:
`1)` need to wait for confirmation in en #13906

Reply:
Confirmed, and I changed to dot as same as the en version.

Reply:
LGTM. Wait for more reviews for such a long PR.